$NetBSD: patch-ad,v 1.4 2013/05/26 18:37:39 wiz Exp $

fix misuse of "bool", should fix build with some compilers, from
Pouya D. Tafti per pkgsrc-users

--- src/song_update.c.orig	2012-06-28 00:46:58.000000000 +0900
+++ src/song_update.c
@@ -187,7 +187,7 @@ song_file_update_inarchive(struct song *
 	if (suffix == NULL)
 		return false;
 
-	plugin = decoder_plugin_from_suffix(suffix, false);
+	plugin = decoder_plugin_from_suffix(suffix, NULL);
 	if (plugin == NULL)
 		return false;
 
